tests/local_load.rs

```diff
@@ -0,0 +1,28 @@
+use std::collections::HashMap;
+use std::path::Path;
+
+use assert_fs::prelude::*;
+
+#[tokio::test]
+async fn it_loads_local_assets() {
+    let origin = assert_fs::TempDir::new().unwrap();
+
+    let mut files = HashMap::new();
+    files.insert("README.md", "# axoasset");
+    files.insert("styles.css", "@import");
+
```

Comment:
could also be 

```rust
let files = HashMap::from_iter([
  ("README.md", "# axoasset"),
  ("styles.css", "@import"),
]);
```

if you want a more declarative style

Reply:
oooh i wasn't familiar but i def prefer, i'll update to this

Reply:
this actually gives me a type error related to async stuff, i may or may not come back to this, but won't include for now